src/data/real/ennreal.lean

```diff
@@ -764,6 +770,9 @@ begin
   rw [← coe_add, ← coe_sub, coe_lt_coe, coe_lt_coe, lt_sub_iff_right],
 end
 
+lemma lt_sub_comm : a < b - c ↔ c < b - a :=
+by rw [lt_sub_iff_add_lt, lt_sub_iff_add_lt, add_comm]
```

Comment:
This could be generalized to `add_comm_group` + the correct `covariant_class` and go in `algebra.order.group` close to `lt_sub_iff_add_lt`. Can you provide the multiplication version as well?

Reply:
`ennreal` is not an additive group. I'm going to add a `has_ordered_sub` instance and remove some of these lemmas later, but I didn't want to mix two refactors in one branch.